README.md

```diff
@@ -34,6 +34,96 @@ load("@build_bazel_rules_nodejs//:defs.bzl", "node_repositories")
 node_repositories(package_json = ["//:package.json"])
 ```
 
+### Installation with a specific supported version of NodejS and Yarn
+
+In your `WORKSPACE` file, use the following:
+
+```python
+git_repository(
+    name = "build_bazel_rules_nodejs",
+    remote = "https://github.com/bazelbuild/rules_nodejs.git",
+    tag = "0.3.1", # check for the latest tag when you install
+)
+
+load("@build_bazel_rules_nodejs//:defs.bzl", "node_repositories")
+
+# NOTE: this rule installs nodejs, npm, and yarn, but does NOT install
+# your npm dependencies into your node_modules folder.
+# You must still run the package manager to do this.
+node_repositories(package_json = ["//:package.json"], node_version = "8.11.1", yarn_version = "1.5.1")
+```
+
+### Installation with a manually specified version of NodeJS and Yarn
+
+If you'd like to use a version of NodeJS and/or Yarn that are not currently supported here, you can manually
+specify those via:
+
+```python
+git_repository(
+    name = "build_bazel_rules_nodejs",
+    remote = "https://github.com/bazelbuild/rules_nodejs.git",
+    tag = "0.3.1", # check for the latest tag when you install
+)
+
+load("@build_bazel_rules_nodejs//:defs.bzl", "node_repositories", "node_download_runtime", "yarn_download")
+
+node_download_runtime(
+    name = "nodejs",
+    version = "8.10.0",
```

Comment:
not sure what we can do about this, but we've seen a request to have the version of npm be free to modify as well given a fixed version of node.

Reply:
They're packaged together, so I imagine this will be difficult.  Deferring that for now.